curriculum/challenges/english/01-responsive-web-design/css-grid/use-grid-column-to-control-spacing.english.md

```diff
@@ -34,8 +34,8 @@ Make the item with the class <code>item5</code> consume the last two columns of
 tests:
   - text: <code>item5</code> class should have a <code>grid-column</code> property.
     testString: assert($('style').text().replace(/\s/g, '').match(/\.item5{.*grid-column:.*}/g));
-  - text: <code>item5</code> class should have a <code>grid-column</code> property which results in the <code>div</code> with the <code>item5</code> consuming the last two columns of the grid.
-    testString: assert(hasCorrectSpacing());
+  - text: <code>item5</code> class should have a <code>grid-column</code> property which results it consuming the last two columns of the grid.
+    testString: const colStart = getComputedStyle($('.item5')[0]).gridColumnStart; const colEnd = getComputedStyle($('.item5')[0]).gridColumnEnd; const result = colStart.toString() + colEnd.toString(); assert(result === "24" || result === "2-1" || result === "2span 2" || result === "2span2" || result === "span 2-1" || result === "-12" || result === "span 2span 2" || result === "span 2auto" || result === "autospan 2");
```

Comment:
To make the code easier to read and make it easier to add other possible results as users find them, why not use the following:
```suggestion
    testString: assert(doesGrid5ConsumeLastTwoGrids());
```
and use my other suggestion in the `Before Test` section

Reply:
Is this really necessary?

Reply:
Necessary?  No, but it makes it easier to read and change the code later.  All of those `||` are unnecessary in my opinion.